neo/Consensus/ConsensusService.cs

```diff
@@ -24,27 +24,25 @@ internal class Timer { public uint Height; public byte ViewNumber; }
         private readonly IConsensusContext context;
         private readonly IActorRef localNode;
         private readonly IActorRef taskManager;
-        private readonly Store store;
         private ICancelable timer_token;
         private DateTime block_received_time;
         private bool started = false;
+
         /// <summary>
         /// This will be cleared every block (so it will not grow out of control, but is used to prevent repeatedly
         /// responding to the same message.
         /// </summary>
         private readonly HashSet<UInt256> knownHashes = new HashSet<UInt256>();
-        private bool isRecovering = false;
```

Comment:
Let's still not move this, since it is only a temporary flag set when processing a recovery message to prevent a primary that is recovering it's own prepare request form setting the timer wrong.

Reply:
Ok, I agree, I'll move it back soon. Just figuring out your last explanation, to see how it first in my state model sketch.